CMakeLists.txt

```diff
@@ -122,6 +122,8 @@ ELSE()
   SET(IGSIO_ZLIB_INCLUDE_DIR "")
 ENDIF()
 
+FIND_PACKAGE(OpenCL REQUIRED)
```

Comment:
Adding OpenCL as a hard dependency is not desirable. Please do the same as in ITK: make it possible to enable/disable GPU acceleration by introducing a new IGSIO_USE_GPU option.

Reply:
Thanks. I definitely don't plan to have it `REQUIRED` in the final version. But while doing development, I rather have it this way because I want to be sure my OpenCL installation (from the CUDA driver, in my case) is working.